test/mason/mason-update-toml-error/tomls/parseloopUnexpectedChar.good

```diff
@@ -0,0 +1 @@
+Unexpected Token on line 1 -> =
```

Comment:
Looking good. I think it'd be nice to make this error message style consistent with Chapel compiler and runtime errors:  `filename:linenumber: error msg`


e.g.

```
parseloopUnexpectedChar.toml:1: Unexpected Token
```

Reply:
For historians: Turns out we only have access to the file channel within the `TOML` module, the TOML module opts for printing just a line number instead.